File: OMS.services/OMS.PatientService/WebApplication1/Models/PatientModel.cs
using System.ComponentModel.DataAnnotations;

namespace WebApplication1.Models
{
    public class PatientModel
    {
        [Key]
        public Guid Patient_Id { get; set; }
        [Required]
        public Guid User_Id { get; set; }


        [Required]
        public string First_Name { get; set; }
        [Required]
        public string Last_Name { get; set; }
        [Required]
        public string Middle_Name { get; set; }
        [Required]
        public string Email { get; set; }
        [Required]
        public int Cell_Phone { get; set; }
        [Required]
        public int Work_Phone { get; set; }
        [Required]
        public int Relationship { get; set; }
        [Required]
        public string Address {  get; set; }
        [Required]
        public string State { get; set; }
        [Required]
        public int NIN {  get; set; }
        [Required]
        public string Sex { get; set; }
        [Required]
        public string DOB { get; set; }



        public string EC_FullName { get; set; }
        public int EC_Cell_Phone { get; set; }
        public string EC_Address { get; set; }



        public DateTime Created { get; set; }
        public DateTime Updated { get; set; }
    }
}

File: OMS.services/OMS.PatientService2/models/patientModel.js
import  { Schema, model } from "mongoose";




const patientSchema = new Schema({
    email: {
        unique: true,
        required: true,
        type: String
    },
    first_Name: {
        required: true,
        type: String
    },
    last_Name: {
        required: true,
        type: String
    },
    middle_Name: {
        required: true,
        type: String
    },
    address: {
        required: true,
        type: String
    },
    cell_Phone: {
        required: true,
        type: String
    }, 
    dob: {
        required: true,
        type: String
    }, 
    nin: {
        unique: true,
        required: true,
        type: String
    }, 
    profile_Url: {
        type: String
    }, 
    relationship: {
        required: true,
        type: String
    }, 
    sex: {
        required: true,
        type: String
    }, 
    state: {
        required: true,
        type: String
    },
    eC_FullName: {
        required: true,
        type: String
    },
    eC_Cell_Phone: {
        required: true,
        type: String
    },
    eC_Address: {
        required: true,
        type: String
    }
}, { timestamps: true });

const Patient = model("Patient", patientSchema);

export default Patient;